packages/integration/lib/src/views/compose/compose_screen.dart

```diff
@@ -477,33 +477,37 @@ class _LMFeedComposeScreenState extends State<LMFeedComposeScreen> {
                     break;
                   case LMMediaType.document:
                     {
-                      mediaWidget = LMFeedDocument(
-                        onRemove: () {
-                          composeBloc.add(
-                            LMFeedComposeRemoveAttachmentEvent(
-                              index: index,
-                            ),
-                          );
-                        },
-                        documentFile: composeBloc.postMedia[index].mediaFile,
-                        style: style?.mediaStyle?.documentStyle?.copyWith(
-                              width: style?.mediaStyle?.documentStyle?.width ??
-                                  MediaQuery.of(context).size.width - 84,
-                            ) ??
-                            LMFeedPostDocumentStyle(
-                              width: screenSize!.width - 84,
-                              height: 90,
-                            ),
-                        size: PostHelper.getFileSizeString(
-                            bytes: composeBloc.postMedia[index].size ?? 0),
+                      mediaWidget = Padding(
+                        padding: const EdgeInsets.symmetric(horizontal: 18.0),
```

Comment:
if we are adding padding across document tile, shouldn't it be customisable as per the clients need?